src/pluginConfig.ts

```diff
@@ -3,9 +3,11 @@
  */
 export interface PluginConfigType {
   ios: {
+    CodePushServerURL: string;
     CodePushDeploymentKey: string;
   };
   android: {
+    CodePushServerURL: string;
```

Comment:
Mark that as optional, not to break existing users who haven't migrated to a personal server yet. We can mark as required after codepush officially sunsets.

Reply:
Done